commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AddAuditLogHandler.java

```diff
@@ -0,0 +1,48 @@
+package org.wildfly.extras.creaper.commands.auditlog;
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.offline.OfflineCommand;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+public final class AddAuditLogHandler implements OnlineCommand, OfflineCommand {
+
+    private final String handlerName;
+
+    public AddAuditLogHandler(String name) {
+        if (name == null) {
+            throw new IllegalArgumentException("Handler name must be specified as non null value");
+        }
+        if (name.isEmpty()) {
+            throw new IllegalArgumentException("Handler name cannot be empty");
+        }
+
+        this.handlerName = name;
+    }
+
+    @Override
+    public void apply(OnlineCommandContext ctx) throws Exception {
+        Operations ops = new Operations(ctx.client);
+        Address handlerAddress = Address.coreService("management")
+                .and("access", "audit")
+                .and("logger", "audit-log")
+                .and("handler", handlerName);
+
+        if (!ops.exists(handlerAddress)) {
```

Comment:
We didn't. I agree that the command should be consistent with the rest. Therefore it should probably have `replaceExisting` field.
On the other hand it only adds a handler with the only one attribute. So basically there are two possible circumstances:
- "new" handler is being written -> OK
- "new" handler is already present in the configuration -> we want to add _exactly the same_ thing (and it must  _always be the exactly same thing_) that already exists in the configuration

It looked to me odd to have `replaceExisting` field here when I wrote this class. Frankly, I still think it doesn't make much sense here. Anyway, did I get right that you insist on having `replaceExisting` everywhere? :-)